cr8/clients.py

```diff
@@ -42,6 +42,10 @@ def _to_http_hosts(hosts: Union[Iterable[str], str]) -> List[str]:
         return r['duration']
 
 
+def plain_or_callable(obj):
+    return obj and callable(obj) and obj() or obj
```

Comment:
I think `obj() if callable(obj) else obj` would be more readable. Or `obj if not callable(obj) else obj()` if you'd prefer that.
And please prefix the method with `_`